coala_quickstart/green_mode/bear_settings.yaml

```diff
@@ -0,0 +1,94 @@
+  GNUIndentBear:
+  # discarding `indent_cli_options` as all options already handled by
+  # other bear settings.
+  CPDBear:
+    language: ['C#', 'C++'] # As currently IMPORTANT_BEAR_LIST
+    # and GREEN_MODE_COMPATIBLE_BEAR_LIST use it for only these 2 languages.
+  CPPCheckBear:
+    enable: ['all', 'warning', 'style', 'performance',
+             'portability', 'information', 'unusedFunction',
+             'missingInclude']
+  PycodestyleBear:
+    pycodestyle_ignore: ['D100', 'D101', 'D102', 'D103', 'D104',
+                        'D105','D106', 'D107',
+                        'D200', 'D201', 'D202', 'D203', 'D204',
+                        'D205', 'D206', 'D207', 'D208', 'D209',
+                        'D210', 'D211', 'D212', 'D213', 'D214',
+                        'D215',
+                        'D300', 'D301', 'D302',
+                        'D400', 'D401', 'D402', 'D403', 'D404',
+                        'D405', 'D406', 'D407', 'D408', 'D409',
+                        'D410', 'D411', 'D412', 'D413', 'D414']
+    pycodestyle_select:  ['D100', 'D101', 'D102', 'D103', 'D104',
+                          'D105','D106', 'D107',
+                          'D200', 'D201', 'D202', 'D203', 'D204',
+                          'D205', 'D206', 'D207', 'D208', 'D209',
+                          'D210', 'D211', 'D212', 'D213', 'D214',
+                          'D215',
+                          'D300', 'D301', 'D302',
+                          'D400', 'D401', 'D402', 'D403', 'D404',
+                          'D405', 'D406', 'D407', 'D408', 'D409',
+                          'D410', 'D411', 'D412', 'D413', 'D414']
+  JSHintBear:
+    es_version: [3, 5, 6]
+    allow_latedef: [True, False, 'nofunc']
+    allow_unused_variables: [True, False, 'vars', 'strict']
+    allow_variable_shadowing: [True, False, 'inner', 'outer']
+    javascript_strictness: [True, False, 'implied', global]
+
+  # The following settings are Type 2 but migrated to Type 3 because
+  # of probable values and lack of efficient algorithms to deduce their
+  # appropriate values from the prohect.
+
+  GNUIndentBear:
+    case_indentation: [0, 2, 4, 8, 1, 3, 5, 6]
+    declaration_indent: [0, 2, 4, 8, 1, 3, 5, 6]
+    brace_indent: [2, 4, 8, 1, 3, 5, 6, 0]
+    indent_size: [4, 2, 8, 1, 3, 5, 6, 0]
+  SpaceConsistencyBear:
+    indent_size: [4, 2, 8, 1, 3, 5, 6, 0]
+
+type2:
+  FilenameBear:
+    filename_prefix: ''
+    filename_suffix: ''
+  LineCountBear:
+    min_lines_per_file: 1
+    max_lines_per_file: 1000
+  GNUIndentBear:
+    max_line_length: 79
+  PycodestyleBear:
+    max_line_length: 79
+
+how_to_guess_these:
+  ClangComplexityBear:
+    cyclomatic_complexity: 8
+  CPDBear:
+    minimum_tokens: 20
+  CPPCleanBear:
+    include_paths: ''
+  JSHintBear:
+    cyclomatic_complexity: None
+    max_depth: None
+
+too_difficult_to_guess_by_file_dict_parsing:
+  JSHintBear:
+    max_parameters: None
+    max_statements: None
+  InvalidLinkBear:
```

Comment:
Some of the settings for this bears are unguessable (like network timeout), the best bet is only to supply some default values and see if they work.